torch/csrc/jit/ir/scope.h

```diff
@@ -140,6 +143,8 @@ struct TORCH_API InlinedCallStack : public c10::intrusive_ptr_target {
 
   // Return callstack as a vector of [Function, SourceRange] pairs.
   std::vector<InlinedCallStackEntry> vec();
+  void setModulePath(std::string path);
+  std::string getModulePath();
```

Comment:
I think we can avoid this all together. Especially in the context that inlinedcallstack knows only about the function it is in and where in that function. In that we added what module that function is in and what instance this particular instance of inlinedcallstack corresponds to. Thus what is all the entry in the callstack put together look like should be outside of this class. So I don't think we need `setModulePath` and `getModulePath` functions. See my later comments as I am typing them.